README.md

```diff
@@ -23,88 +22,101 @@ applications yield the processor and transported using RDMA-like operations, res
-# please perform the following step.
-git clone https://github.com/ovis-hpc/sos.git -b SOS-4
-```
+To clone the source code, go to https://github/com/ovis-hpc/ovis, and click
+one the "Code" button. Or use the following command:
 
-# Dependencies
+```git clone https://github.com/ovis-hpc/ovis.git -b OVIS-4```
 
-* autoconf (>=2.63), automake, libtool
-* glib2
+## Build Dependencies
+
+* autoconf (>=2.63)
+* automake
+* libtool
+* make
+* bison
+* flex
+* gettext development package
+* libevent developmetn package
 * libreadline
-* openssl Development library for OVIS, LDMS Authentication
-* libmunge for Munge LDMS Authentication plugin
-* For LDMS Python Interface:
-	* Python-3.6.
-	* Cython 0.25.
-* doxygen if you want to build OVIS documentation.
-* Some LDMS plug-ins have dependency on additional libraries.
+* openssl development library (for OVIS, LDMS Authentication)
+* libmunge (for Munge LDMS Authentication plugin)
+* Python >= 3.6 and Cython >= 0.25 (for the LDMS Python API and ldmsd_controller)
+* doxygen (for the OVIS documentation)
+
+Some LDMS plug-ins have dependencies on additional libraries.
+
 For cray-related LDMS sampler plug-in dependencies, please see the man page of the
 plug-in in `ldms/man/`.
 
+### RHEL7/CentOS7 dependencies
+
+RHEL7/CentOS7 systems will require a the following packages at a minimum:
 
-# Building OVIS
+* autoconf
+* automake\
+* libtool
+* make
+* bison
+* flex
+* gettext-devel
+* libevent-devel
+* openssl-devel
+
+Additionally, the Python API and the ldmsd_controller command require Python and Cython.
+One way to obtain those packages is from EPEL (install the epel-release package, and
+then "yum update"). The packages from EPEL are:
+
+* python3-devel
+* python36-Cython
+
+## Compling the code
+
+If you are interested in storing LDMS data in SOS, then first
+follow the instructions at https://github.com/ovis-hpc/sos to obtain,
+build, and install SOS before proceding.
 
 ```sh
-	# If you're interested in SOS-dependent storage plugin, please install SOS first as follows
-	# otherwise, please skip to `Install ovis-ldms` below
-	cd sos
-	./autogen.sh
-	mkdir <build directory>
-	cd <build directory>
-	../configure --prefix<installed path> [options]
-	make
-	make install
-	cd ../../
-	
-	# Install ovis-ldms
-	cd ovis
-	./autogen.sh
-	mkdir <build directory>
-	cd <build directory>
-	../configure --prefix=<installed path> [options]
-	#   also add --with-sos=<installed path> if SOS is required
+	cd <ovis source directory>
+	sh autogen.sh
+	./configure [--prefix=<installation prefix>] [other options]
 	make
 	make install
 ```
 
-# Supported hardware
+Run ```configure --help``` for a full list of configure options.
+
+# Supported systems
 
 * Ubuntu and friends
 * CentOS and friends
 * Cray XE6, Cray XK, Cray XC
 
 # Unsupported features
-LDMS sampler plugins
+
+The following LDMS sampler plugins are considered unsupported. Use are your own risk:
 * perfevent sampler
 * papi sampler
```

Comment:
Do you mean the hweventpapi sampler? The papi sampler is supported.

Reply:
I don't know on that one. That was left over from the previous version. I'll assume hweventpapi is the right one and make that change.